phys2bids/phys2bids.py

```diff
@@ -265,15 +265,19 @@ def phys2bids(filename, info=False, indir='.', outdir='.', heur_file=None,
 
     # Create trigger plot. If possible, to have multiple outputs in the same
     # place, adds sub and ses label.
-    LGR.info('Plot trigger')
-    plot_path = os.path.join(outdir,
-                             os.path.splitext(os.path.basename(filename))[0])
-    if sub:
-        plot_path += f'_sub-{sub}'
-    if ses:
-        plot_path += f'_ses-{ses}'
-    viz.plot_trigger(phys_in.timeseries[0], phys_in.timeseries[chtrig],
-                     plot_path, tr, thr, num_timepoints_expected)
+    if tr != 1 or num_timepoints_expected != 0:
```

Comment:
What if the TR is actually 1?

Reply:
I think is highly unlikely and also the moment you add the -npt i won't matter. In the logger it is actually recommended to add both

Reply:
But if you guys think if can become an issue, we can use only -ntp

Reply:
Why don't we set the default TR (in the CLI) to 0 instead?

Reply:
good idea, let me change it

Reply:
I am gonna check is not a problem in other things, if we introduce it as zero

Reply:
the program doesn't stop with tr=0 as default, let me push to check travis

Reply:
The whole program shouldn't stop with default TR, be it one or zero!

Reply:
seems like test are fine, I think this conversation can be resolved